docs/.gitignore

```diff
@@ -1 +1 @@
-*/_build
\ No newline at end of file
+**/_build
```

Comment:
what's the point of replacing * with ** here?

Reply:
** means any folder depth, * means one level.